app/src/main/res/raw/vn_sdk_config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Should we add comments in here? I think that would break the parsing though

Reply:
Yeah that’s what I was worried about. But also, I don’t think it is necessary because nobody should need to even look at this file.

Reply:
Yeah, no one's going to use the file but I don't think we should be explaining a config file here anyway. If they are being sent a config file to use, we should be telling them about it then.